core/domain/stats_jobs_one_off.py

```diff
@@ -34,6 +36,46 @@
 ])
 
 
+class RemoveBadPlaythroughsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job for removing all playthroughs created between the July 2018
+    release and the August 2018 release.
+    """
+
+    AUGUST_RELEASE_DATETIME = datetime.datetime(2018, 9, 3, 7, 47, 0, 0)
```

Comment:
It's OK to remove a few more extra playthroughs from after the actual datetime, just to make absolutely sure that everything that remains is valid. So if you want to advance the date by a couple of days I think that's fine.